tests/Integration/Commands/New-SqlDscLogin.Integration.Tests.ps1

```diff
@@ -0,0 +1,237 @@
+            }
+
+            It 'Should verify the SQL Server login was created' {
+                Test-SqlDscIsLogin -ServerObject $script:serverObject -Name $script:testSqlLoginName | Should -BeTrue
+            }
+
+            It 'Should verify the login type is SqlLogin' {
+                $loginObject = $script:serverObject.Logins[$script:testSqlLoginName]
+                $loginObject.LoginType | Should -Be 'SqlLogin'
+            }
+
+            It 'Should verify the default database is set correctly' {
+                $loginObject = $script:serverObject.Logins[$script:testSqlLoginName]
+                $loginObject.DefaultDatabase | Should -Be 'master'
+            }
+
+            It 'Should create a SQL Server login with custom default database' {
+                $customLoginName = 'IntegrationTestCustomDb'
+
+                try
+                {
+                    { New-SqlDscLogin -ServerObject $script:serverObject -Name $customLoginName -SqlLogin -SecurePassword $script:testPassword -DefaultDatabase 'tempdb' -Confirm:$false } | Should -Not -Throw
+
+                    $loginObject = $script:serverObject.Logins[$customLoginName]
+                    $loginObject.DefaultDatabase | Should -Be 'tempdb'
+                }
+                finally
+                {
+                    # Clean up
+                    if (Test-SqlDscIsLogin -ServerObject $script:serverObject -Name $customLoginName)
+                    {
+                        $script:serverObject.Logins[$customLoginName].Drop()
+                    }
+                }
+            }
+
+            It 'Should create a SQL Server login with PassThru parameter' {
+                $passthroughLoginName = 'IntegrationTestPassThru'
+
+                try
+                {
+                    $result = New-SqlDscLogin -ServerObject $script:serverObject -Name $passthroughLoginName -SqlLogin -SecurePassword $script:testPassword -PassThru -Confirm:$false
+
+                    $result | Should -Not -BeNullOrEmpty
+                    $result.Name | Should -Be $passthroughLoginName
+                    $result.LoginType | Should -Be 'SqlLogin'
+                }
+                finally
+                {
+                    # Clean up
+                    if (Test-SqlDscIsLogin -ServerObject $script:serverObject -Name $passthroughLoginName)
+                    {
+                        $script:serverObject.Logins[$passthroughLoginName].Drop()
+                    }
+                }
+            }
+
+            It 'Should create a disabled SQL Server login' {
+                $disabledLoginName = 'IntegrationTestDisabled'
+
+                try
+                {
+                    { New-SqlDscLogin -ServerObject $script:serverObject -Name $disabledLoginName -SqlLogin -SecurePassword $script:testPassword -Disabled -Confirm:$false } | Should -Not -Throw
+
+                    $loginObject = $script:serverObject.Logins[$disabledLoginName]
```

Comment:
Add ned command `Test-SqlDscIsLoginEnabled` (that also uses `Test-SqlDscIsLogin` to verify exist).